File: src/main/java/edu/kit/iti/formal/stvs/model/examples/ElevatorExample.java
package edu.kit.iti.formal.stvs.model.examples;

/**
 * @author Alexander Weigl
 * @version 1 (01.04.17)
 */
public final class ElevatorExample extends Example {
    public ElevatorExample() {
        name = "Elevator";
        description = "Simple Demonstration";
        htmlHelp = getClass().getResource("elevator.html").toExternalForm();
        sessionFile = getClass().getResource("elevator.xml");
    }
}

File: stvs/src/main/kotlin/edu/kit/iti/formal/stvs/model/examples/ElevatorExample.kt
package edu.kit.iti.formal.stvs.model.examples

/**
 * @author Alexander Weigl
 * @version 1 (01.04.17)
 */
class ElevatorExample : Example() {
    init {
        name = "Elevator"
        description = "Simple Demonstration"
        htmlHelp = javaClass.getResource("elevator.html").toExternalForm()
        sessionFile = javaClass.getResource("elevator.xml")
    }
}
